src/log.rs

```diff
@@ -114,27 +114,28 @@ fn to_log_flags(fatal: bool, recursion: bool) -> u32 {
 }
 
 #[cfg(any(feature = "v2_46", feature = "dox"))]
-pub fn log_set_handler<P: Fn(&str, LogLevel, &str) + Send + Sync + 'static>(
-    log_domain: &str,
+pub fn log_set_handler<P: Fn(Option<&str>, LogLevel, &str) + Send + Sync + 'static>(
```

Comment:
No, the domain that is passed to the callback is always not `NULL` :)

```suggestion
pub fn log_set_handler<P: Fn(&str, LogLevel, &str) + Send + Sync + 'static>(
```